TROUBLESHOOTING.md

```diff
@@ -0,0 +1,71 @@
+#Troubleshooting
+
+## Log Files
+
+Effectively debugging software requires as much relevant information as can be
+obtained. To assist the ownCloud support personnel, please try to provide as
+many relevant logs as possible. Log output can help with tracking down
+problems and, if you report a bug, log output can help to resolve an issue more
+quickly.
+
+Here you have a description about how to get relevant information to be handled in case of any issue. You can ship this information via mail (android-app@owncloud.com), creating a new issue in the [open GitHub ownCloud Android repository](https://github.com/owncloud/android), or in [public ownCloud forum](https://central.owncloud.org/).
+
+### Capturing App Debug Logs
+
+You will find the Logs in the Settings view of the Android app. If the Logs option is not visible, tap five times on the build number (bottom of the Settings view), and Logs will be an option inside the "More" section.
+
+When you have reproduce the issue you want to address, the "Send History" button will automatically add the logs to a new mail message.
+
+1. Delete History
```

Comment:
Maybe a description before explaining what are these steps for? Something like:

_When you have reproduced the issue you want to address, the "Send History" button will automatically add the logs to a new mail message. Please follow the next steps to send us your logs_

Reply:
good idea! added.